Sources/Private/CoreAnimation/Layers/LayerModel+makeAnimationLayer.swift

```diff
@@ -51,6 +51,9 @@ extension LayerModel {
     case (.null, _):
       return TransformLayer(layerModel: self)
 
+    case (.unknown, _):
+      return nil
```

Comment:
This fixes an issue where a layer with an unknown type would log a compatibility error and fall back to the main thread rendering engine, even though the main thread engine wouldn't be able to handle it either

Reply:
Should we add a this comment to the code?